drivers/i2c/Kconfig.stm32

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig I2C_STM32
```

Comment:
STM32L0 series was added recently and it will be good to cleanup its Kconfig (arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.series)